gravitee-apim-console-webui/src/management/api/api-traffic-v4/runtime-logs/components/api-runtime-logs-quick-filters/api-runtime-logs-quick-filters.component.html

```diff
@@ -74,16 +74,16 @@
             class="gio-badge-neutral quick-filters__chip"
             *ngIf="filter.value && defaultFilters[filter.key] !== filter.value"
             [removable]="true"
-            (removed)="removeFilter(filter)"
-            [matTooltip]="filter.value | chipValue"
+            (removed)="removeFilter($any(filter))"
```

Comment:
can you point me some doc that can explain this `$any` thing? 🙏  
I suppose it used to cast to `any` 🤔

Reply:
Yes same question from @Okhelifi.

The keyvalue pipe seems to lose its typing and we end up with an unknown. 
and it is not possible to type directly in html (exception of $any)

We could add the same thing with a public method that forces the type. but for this PR i went for the simplest

Reply:
so here we know we're losing the typing. We didn't have it before but now it's explicit. 

If you want to look, maybe there's another solution, but I didn't find it in a few minutes of searching.

Reply:
ho maybe i'me wrong for this line hum hum. i've doute now. 
because I ended up applying this to each keyvalue pipe 🤖

Reply:
![image](https://github.com/gravitee-io/gravitee-api-management/assets/4974420/75310e46-0b44-47bb-ad74-76a749dcbb6b)
any idea to fix ?

Reply:
hum, 
fixed with `removeFilter(removedFilter: KeyValue<string, unknown>) {`
we dont need value type in this case